.github/workflows/ci_macos.yml

```
@@ -0,0 +1,119 @@
+name: CI on macOS
+on:
+  push:
+    branches:
+      # Push events to branches matching refs/heads/master
+      - 'master'
+  pull_request:
+
+env:
+  CMAKE_VERSION: 3.8.2
+  SEQAN3_NO_VERSION_CHECK: 1
+  TZ: Europe/Berlin
+
+defaults:
+  run:
+    shell: bash -ex {0}
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: macos-10.15
+    timeout-minutes: 120
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - name: "gcc10"
+            os: macos-10.15
+            cxx: "g++-10"
+            cc: "gcc-10"
+            build_type: Release
+
+          - name: "gcc9 (c++2a)"
+            os: macos-10.15
+            cxx: "g++-9"
+            cc: "gcc-9"
+            build_type: Release
+            cxx_flags: "-std=c++2a"
+
+          - name: "gcc8"
+            os: macos-10.15
+            cxx: "g++-8"
+            cc: "gcc-8"
+            build_type: Release
+
+          - name: "gcc7"
+            os: macos-10.15
+            cxx: "g++-7"
+            cc: "gcc-7"
+            build_type: Release
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: src
+          fetch-depth: 2
+          submodules: recursive
+
+      - name: Configure Homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Install CMake
+        run: bash ./src/lib/seqan3/.github/workflows/scripts/install_cmake.sh
```

```suggestion
        run: bash ./src/lib/seqan3/.github/workflows/scripts/install_cmake.sh ${cmake_version}
```

not needed now, but I would prefer if we pass the version explicitly :)